src/functions/function-command/invoke-function-def.ts

```diff
@@ -0,0 +1,243 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+
+import { WizardPageFieldDefinition, WizardPageSectionDefinition } from '@redhat-developer/vscode-wizard';
+import { invokeFunctionID } from '../webview-id';
+
+interface Select {
+  key: string;
+  label: string;
+}
+
+const templateSelect: Array<Select> = [
+  { key: 'http', label: 'http' },
+  { key: 'cloudevents', label: 'cloudevents' },
+];
+
+export const invokeInstance: WizardPageFieldDefinition = {
+  id: invokeFunctionID.invoke_Instance,
+  label: 'Invoke Instance',
+  type: 'radio',
+  initialValue: 'Local',
+  properties: {
+    options: ['Local', 'Remote'],
+  },
+};
+
+export const invokeNamespace: WizardPageFieldDefinition = {
+  id: invokeFunctionID.invoke_namespace,
+  label: 'Namespace',
+  placeholder: 'Provide namespace',
+  type: 'textbox',
+};
+
+export const invokeID: WizardPageFieldDefinition = {
+  id: invokeFunctionID.invoke_ID,
+  label: 'ID',
+  placeholder: 'Automatically generated. (optional)',
+  type: 'textbox',
+};
+
+export const invokeUrl: WizardPageSectionDefinition = {
+  id: invokeFunctionID.invoke_Url_Def,
+  label: 'Function instance to invoke',
+  childFields: [
+    {
+      id: invokeFunctionID.invoke_Url_Check,
+      label: 'Target this custom URL when invoking the function',
+      type: 'checkbox',
+    },
+    {
+      id: invokeFunctionID.invoke_Url,
+      label: 'URL',
+      placeholder: 'Target custom URL when invoking the function. (optional)',
+      type: 'textbox',
+      properties: { disabled: true },
+    },
+  ],
+};
+
+// export const invokeUrl: WizardPageFieldDefinition = {
+//   id: invokeFunctionID.invoke_Url,
+//   label: 'URL',
+//   placeholder: 'Target custom URL when invoking the function. (optional)',
+//   type: 'textbox',
+// };
+
+export const invokePath: WizardPageFieldDefinition = {
+  id: invokeFunctionID.invoke_path,
+  label: 'Path',
+  placeholder: 'Path which has the instance invoked.',
+  type: 'textbox',
+  initialState: {
+    enabled: false,
```

Comment:
This is why the Path label is disabled. This should have a `readonly: true` state. The wizard doesn't have it?

Reply:
https://github.com/redhat-developer/vscode-wizard/blob/master/src/StandardWizardPageRenderer.ts#L125 `wrapHTMLField` taking the `disabled value` and creating `dif with disabled`

Reply:
What does `dif with disabled` mean here?

Reply:
The disabled label Path is not looking good on the UI. And is there a way to remove the blue colours from all the label texts?

Reply:
label color has been changed in UI.